### PR TITLE
the most minor changes

### PR DESCRIPTION
**Pull request by alexandergmzx**

### CODE REVIEW

**Comment by Israel-SanchezH** [comment]
Looks great!!!, it is more descriptive

File: README.md [hunk 1]
```diff
@@ -3,4 +3,4 @@
 Este es un ejemplo de como podemos trabajar de manera colaborativa con git.
 
 Update 2 (Homework 3)
-Activity 3 from second module, this is made to practice the collaboration with our teammates.
+Activity 3 from second module, this is made to practice the collaboration with humans.
```
Comment (mocadev1) [inline README.md]: Nice correction, could you change to "human being"?.

File: src/app.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
 /*TODO: Set app headers, including the CORS */
-# THE AP
\ No newline at end of file
+# THE AP IS HERE!!
```
Comment (mocadev1) [inline src/app.md]: Nice header!

**Review by mocadev1 — CHANGES_REQUESTED** [review]

File: src/app.md (hunk 2 above)
Comment (Israel-SanchezH) [reply src/app.md]: There is a mistake in the word (AP). it should be (APP), please update it

File: README.md (hunk 1 above)
Comment (Israel-SanchezH) [reply README.md]: Looks great!!! it is more descriptive

**Review by Israel-SanchezH — CHANGES_REQUESTED** [review]

**Pull request merged**